data/World/Fire Temple.json

```diff
@@ -13,7 +13,9 @@
                 ((Small_Key_Fire_Temple, 8) or not keysanity) and can_use(Megaton_Hammer)",
             "Fairy Pot": "
                 has_bottle and (can_use(Hover_Boots) or can_use(Hookshot)) and
-                (logic_fewer_tunic_requirements or can_use(Goron_Tunic))"
+                (logic_fewer_tunic_requirements or can_use(Goron_Tunic))",
+            "Fire Temple Near Boss Pot 1": "(can_use(Hover_Boots) or can_use(Hookshot)) and (logic_fewer_tunic_requirements or can_use(Goron_Tunic))",
+            "Fire Temple Near Boss Pot 2": "(can_use(Hover_Boots) or can_use(Hookshot)) and (logic_fewer_tunic_requirements or can_use(Goron_Tunic))"
```

Comment:
Boomerang trick throws can be used here as well on at least one of the pots. Seems painful to actually perform.